dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -214,7 +215,30 @@ void testStarringNotebook() {
         notebook = workflowsApi.getWorkflow(notebookID, "");
         assertEquals(0, notebook.getStarredUsers().size());
     }
+    @Test
+    void testNotebookRSSFeedAndSitemap() {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
```

Comment:
This pre-dates this PR, but this class inherits from `BaseIT` which has 
`public void resetDBBetweenTests()` which resets the DB with simple test data. 
Then if I'm not mistaken, every test in here uses `CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);` which may mean that the DB is being reset twice for each test. 

Maybe you should just override `resetDBBetweenTests` to reset with `cleanStatePrivate2`